contrib_bots/lib/EncryptBot.py

```diff
@@ -0,0 +1,54 @@
+def encrypt(text):
+    #This is where the actual ROT13 is applied
+    textlist = list(text)
+    newtext = ''
+    firsthalf = list('abcdefghijklmABCDEFGHIJKLM')
+    lasthalf = list('nopqrstuvwxyzNOPQRSTUVWXYZ')
```

Comment:
You don't need to turn a string into list; string can already be considered an immutable list.